samples/apps/copilot-chat-app/webapi/Skills/utils.cs

```diff
@@ -24,4 +24,14 @@ internal static SKContext CopyContextWithVariablesClone(SKContext context)
             context.CancellationToken
         );
     }
+
+    /// <summary>
+    /// Estimate the number of tokens in a string.
+    /// TODO: This is a very naive implementation. We should use the new implementation that is available in the kernel.
```

Comment:
So why aren't we?

Reply:
Although the kernel implementation has similar logic, the estimation parameter is not configurable.